### ISSUE TITLE
error: build of ‘/nix/store/cpkhlkq5gzm1gz9sgawzqxl49hfpqyy3-git-2.3.3.drv’ failed

### ISSUE THREAD

**Issue by nomosyn** [NixOS/nixpkgs#6912]
![capture d ecran 2015-03-20 a 19 44 18](https://cloud.githubusercontent.com/assets/2693150/6758603/8c764e60-cf39-11e4-9e8a-ecab4b2e8136.png)

- `NIX_PATH` points to a clone of https://github.com/NixOS/nixpkgs.git.

- did `$ nix-env -u`

- got the following:

```
[... lots of stuff ...]
building path(s) ‘/nix/store/60266vr5ypahajlnsc1465y0w0c1p619-coreutils-8.23’
unpacking sources
unpacking source archive /nix/store/0cijybc2m6n9g80phq584fb3sgdb3ldi-coreutils-8.23.tar.xz
source root is coreutils-8.23
patching sources
sed: 1: "2i echo Skipping dd spa ...": command i expects \ followed by text
builder for ‘/nix/store/3jymh1pcw8bjdyzgy1xlwf8hlcfxkzis-coreutils-8.23.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/dl26yfmsgr943hzda4pcdi5cszfc4gmj-stdenv-darwin-boot-3.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/cpkhlkq5gzm1gz9sgawzqxl49hfpqyy3-git-2.3.3.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/cpkhlkq5gzm1gz9sgawzqxl49hfpqyy3-git-2.3.3.drv’ failed
```

**Comment by nomosyn** [comment]
I did a `git pull` inside de nixpkgs clone, pb does not show up anymore but a new one does #6934.
I guess that this issue can be closed...

**Comment by giogadi** [comment]
I don't see this issue anymore either, I only see #6934. @nomosyn can you please close this issue?

**Comment by edolstra** [comment]
This was apparently fixed by 68647a4de5afa4a5743e262bf62b0d7048e08a95.
